src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="images/remove-distractions.png" alt="Two screenshots on mobile 
+  showing progress through johnlewis.com checkout. Search, navigation and other distractions are 
+  removed.">
+  <figcaption class="w-figcaption">Search, navigation and other distractions removed for checkout.</figcaption>
+</figure>
+
+Keep the journey focused. This is not the time to tempt users to do something else! 
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="images/free-stickers.jpg" alt="Screenshot of checkout page on 
+  mobile showing distracting promotion for FREE STICKERS." width="350">
+  <figcaption class="w-figcaption">Don't distract customers from completing their purchase.</figcaption>
+</figure>
+
+For returning users you can simplify the checkout flow even more, by hiding data they don't need to 
+see. For example: display the delivery address in plain text (not in a form) and allow users to 
+change it via a link. 
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="images/returning-user.png" alt="Screenshot of 'Review order' 
+  section of checkout page, showing text in plain text, with links to change delivery address, 
+  payment method and billing address, which are not displayed." width="450">
+  <figcaption class="w-figcaption">Hide data customers don't need to see.</figcaption>
+</figure>
+
+## Make it easy to enter name and address {: #address-forms}
+
+### Only ask for the data you need {: #unneeded-data}
+
+Before you start coding your name and address forms, make sure to understand what data is required. 
+Don't ask for data you don't need! The simplest way to reduce form complexity is to remove unnecessary 
+fields. That's also good for customer privacy and can reduce back-end data cost and liability.
+
+### Use a single name input {: #single-name-input}
+
+Allow your users to enter their name using a single input, unless you have a good reason for 
+separately storing given names, family names, honorifics, or other name parts. Using a single name 
+input makes forms less complex, enables cut-and-paste, and makes autofill simpler.
+
+In particular, unless you have good reason not to, don't bother adding a separate input for a 
+prefix or title (like Mrs, Dr or Lord). Users can type that in with their name if they want to. Also, 
+`honorific-prefix` autocomplete currently doesn't work in most browsers, and so adding a field for 
+name prefix or title will break the address form autofill experience for most users.
+
+### Enable name autofill 
+
+Use `name` for a full name:
+
+```html
+<input autocomplete="name" ...>
+```
+
+If you really do have a good reason to split out name parts, make sure to to use appropriate 
+autocomplete values:
+
+* `honorific-prefix`
+* `given-name`
+* `nickname`
+* `additional-name-initial`
+* `additional-name`
+* `family-name`
+* `honorific-suffix`
+
+### Allow international names {: #unicode-matching}
```

Comment:
One additional paragraph worthy tip:  
It's great to have phonetic name fields for a Japanese name. It helps customer support to call out the customer's name on the phone.

Reply:
Ah! Thank you. Added an aside.